src/main/proto/wfa/measurement/api/v2alpha/protocol_config.proto

```diff
@@ -0,0 +1,75 @@
+// Copyright 2021 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package wfa.measurement.api.v2alpha;
+
+import "wfa/measurement/api/v2alpha/differential_privacy.proto";
+
+option java_package = "org.wfanet.measurement.api.v2alpha";
+option java_multiple_files = true;
+option java_outer_classname = "ProtocolConfigProto";
+
+// Resource message for a multi-party computation (MPC) protocol. Immutable.
+message ProtocolConfig {
+  message Key {
+    string protocol_config_id = 1;
+  }
+  // Resource key.
+  Key key = 1;
+
+  // Type of measurement.
+  //
+  // The values should correspond to the fields in the `measurement_type` Oneof
+  // in `MeasurementSpec`.
+  enum MeasurementType {
+    MEASUREMENT_TYPE_UNSPECIFIED = 0;
+    REACH_AND_FREQUENCY = 1;
+  }
+  // The type of measurement that this protocol computes. Required. Immutable.
+  MeasurementType measurement_type = 2;
+
+  // Configuration for the Liquid Legions v2 R/F protocol.
+  message LiquidLegionsV2 {
+    // Parameters for sketch. Required.  Immutable.
+    LiquidLegionsSketchParams sketch_params = 1;
+
+    // Differential privacy parameters for data provider noise registers.
+    // Required.  Immutable.
+    //
+    // Each of these registers contains a well-known constant register ID, and
+    // arbitrary key and count values.
+    DifferentialPrivacyParams data_provider_noise = 2;
+  }
+  // Configuration for the specific protocol. Required. Immutable.
+  oneof protocol {
+    // Liquid Legions v2 config.
+    //
+    // May only be set when `measurement_type` is `REACH_AND_FREQUENCY`.
+    LiquidLegionsV2 liquid_legions_v2 = 3;
+  }
+}
+
+// Parameters for a Liquid Legions sketch.
+message LiquidLegionsSketchParams {
+  // The decay rate of the Liquid Legions sketch. Required.
+  double decay_rate = 1;
```

Comment:
See other questions about which party or who specifies this and how do we ensure all parties agree?

Reply:
I think something went wrong in merging -- this definitely shouldn't be part of this PR. This is from a different PR already in the repo.